site.json

```diff
@@ -2,7 +2,8 @@
   "title": "Node.js",
   "description": "Node.js® is a JavaScript runtime built on Chrome's V8 JavaScript engine.",
   "favicon": "/static/images/favicons/favicon.png",
-  "accentColor": "#333",
+  "lightAccentColor": "#C5E5B4",
```

Comment:
Please update the TypeScript definitions of site.json!

Reply:
Done !